docs/UserGuide.md

```diff
@@ -135,6 +135,58 @@ The suggested command will be automatically typed into the command bar.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## Friends
+
+FriendDex allows you to store certain information about your friends, so you don't have to.
+The information below provides further description about your friends' details you can store.
+
+Further description on how to add/modify these fields are provided in the [Features](#features) section.
+
+### Name
+
+Your friend's name, so you don't have those embarrassing moments where you forget their name.
+
+This field is what uniquely identifies each friend in FriendDex.
+
+### Phone Number
+
+Your friend's phone number, so you don't have to remember their phone number everytime you call.
+
+This field should only contain numbers and should at least be 3 digits long.
+
+### Address
+
+Your friend's home address so that you don't have to remember their address whenever you want to hang out with them.
+
+This field can take any values, but it should not be blank.
```

Comment:
Can add in "It is okay for two friends to share the same address. They could be siblings stored in FriendDex."